README.md

```diff
@@ -28,9 +28,8 @@ Habitat-Sim is typically used with
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/facebookresearch/habitat-sim/)
 
 [![Habitat Demo](https://img.shields.io/static/v1?label=WebGL&message=Try%20AI%20Habitat%20In%20Your%20Browser%20&color=blue&logo=webgl&labelColor=%23990000&style=for-the-badge&link=https://aihabitat.org/demo)](https://aihabitat.org/demo)
-<p align="center">
-  <img src="docs/images/habitat2_small.mp4" height="400">
-</p>
+
+https://user-images.githubusercontent.com/2941091/126080914-36dc8045-01d4-4a68-8c2e-74d0bca1b9b8.mp4
```

Comment:
Hold on does this work? This will just add a link won't it?

Reply:
No, it inserts a viewer. See here: https://stackoverflow.com/questions/4279611/how-to-embed-a-video-into-github-readme-md

I verified by previewing on github.